src/components/Page/Dependency/Dependency.tsx

```diff
@@ -2,23 +2,22 @@ import css from './Dependency.module.css'
 import Header from '../../UI/Header/Header'
 import { IDependencyPage } from '../../../model'
 import InformationTabs, { TABS } from '../../UI/InformationTabs/InformationTabs'
-import ApplicabilityEvidence from './ApplicabilityEvidence'
 export interface Props {
 	data: IDependencyPage
 }
 
 export default function Dependency(props: Props): JSX.Element {
-	const showApplicabilityEvidence =
-		props.data.cve?.applicableData?.evidence ?? props.data.cve?.applicableData?.searchTarget
 	return (
 		<div className={css.PageContainer}>
 			<Header pageData={props.data} text={props.data.cve?.id ? props.data.cve.id : props.data.id} />
-			{props.data.cve?.applicableData && showApplicabilityEvidence && (
-				<ApplicabilityEvidence data={props.data.cve.applicableData} />
-			)}
 			<InformationTabs
 				data={props.data}
-				tabs={[TABS.WHAT_CAN_I_DO.key, TABS.CVE_INFORMATION.key, TABS.IMPACT_GRAPH.key]}
+				tabs={[
+					TABS.WHAT_CAN_I_DO.key,
+					TABS.CVE_INFORMATION.key,
+					TABS.IMPACT_GRAPH.key,
+					TABS.CONTEXTUAL_ANALYSIS.key
+				]}
```

Comment:
what if not JAS scan (i.e not entitled), should we also show the new tab?

Reply:
I don't want to add JAS logic in the front so I'll let the back do the logic. 
Added:
	const showContextualAnalysisTab =
		showApplicabilityEvidence && props.tabs.includes(TABS.CONTEXTUAL_ANALYSIS.key)
If not entitled, show applicability will be false and we won't show the tab